lib/tasks/send_survey.rake

```diff
@@ -0,0 +1,27 @@
+namespace :survey do
```

Comment:
I think it wants you to add '# frozen_string_literal: true`' (or false) to the top of the file. I'm not sure whether we need it. This will make strings immutable, and is default in Ruby 3, it just wants you to be explicit about whether you want mutability, I guess?